packages/dds/sequence/src/defaultMap.ts

```diff
@@ -117,7 +94,37 @@ export class DefaultMap<T> {
 	/**
 	 * Mapping of op types to message handlers.
 	 */
-	private readonly messageHandlers: ReadonlyMap<string, IMapMessageHandler> = new Map();
+	private readonly messageHandler = {
```

Comment:
this is just copy pasted from below, and removed from the dictionary, as it always had only one entry, this one